src/Components/WebAssembly/testassets/Wasm.Authentication.Client/wwwroot/seleniumworkaround.js

```diff
@@ -0,0 +1,24 @@
+(function () {
+  const logs = [];
+
+  console.defaultLog = console.log.bind(console);
+  console.log = function () {
+    console.defaultLog.apply(console, arguments);
+    logs.push(Array.from(arguments).join('\n'));
+  }
+
+  console.defaultError = console.error.bind(console);
+  console.error = function () {
+    console.defaultError.apply(console, arguments);
+    logs.push(Array.from(arguments).join('\n'));
+  }
+
+  console.defaultWarn = console.warn.bind(console);
+  console.warn = function () {
+    console.defaultWarn.apply(console, arguments);
+    logs.push(Array.from(arguments).join('\n'));
+  }
+
+  window.getBrowserLogs = () => logs;
```

Comment:
Nice! I think we need to be able to clear the logs out as well for some test scenarios (thinking about the WebAssemblyConsoleLogger tests).

Reply:
I haven't re-enabled tests that rely on logging as yet. It's definitely something we can look at next